mlir/test/Dialect/Transform/include/test-interpreter-invalid.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: mlir-opt %s --verify-diagnostics
+
+module attributes {transform.with_named_sequence} {
+  transform.named_sequence private @private_helper(%arg0: !transform.any_op {transform.readonly}) {
+    // expected-error @below {{expected ','}}
+    transform.test_print_remark_at_operand %arg0 "message" : !transform.any_op
```

Comment:
How about changing the message to be explicit what's wrong with the syntax? 

```suggestion
    transform.test_print_remark_at_operand %arg0 "should have ',' prior to this" : !transform.any_op
```

Reply:
This is an error produced by the parser. This patch only makes sure we exit gracefully.

Reply:
I thought it might be better to make what's wrong with this IR explicitly for the readability. But anyway, it's trivial.

Reply:
Ah i reread what you said earlier. My bad, i will fix it before landing. Thanks!